package/python3-obs-img-utils.spec

```diff
@@ -27,6 +27,7 @@ Url:            https://github.com/SUSE-Enceladus/obs-img-utils
 Source:         https://files.pythonhosted.org/packages/source/o/obs-img-utils/obs-img-utils-%{version}.tar.gz
 BuildRequires:  python3-devel
 BuildRequires:  python3-setuptools
+BuildRequires:  python3-rpm-macros
```

Comment:
Please use `python-rpm-macros`. `python3-rpm-macros` is just a `Provides` and as such if someone searches for `python3-rpm-macros` in the Build Service nothing will show up which will lead to an unnecessary goose chase.

Reply:
Rebased